src/main/g8/build.sbt

```diff
@@ -20,6 +20,10 @@ organization := "$package$"
 
 version := "0.1.0-SNAPSHOT"
 
+// if you need a specific version of Daffodil, you can uncomment and edit
+// this line to refer to a specific daffodil version.
+// daffodilVersion := "3.10.0"
```

Comment:
Thoughts on removing this? I try to avoid adding specific versions to templates, even as examples, just because they never get updated and quickly look outdated. Plus, the link below details all the available settings, and daffodilVersion is the version one discussed.

Reply:
A fake number like 00.00.00 would avoid the "dated look" issue. 

This comes up so often I think it's worth having it.